include/envoy/network/connection_handler.h

```diff
@@ -68,9 +68,32 @@ class ConnectionHandler {
    * after they have been temporarily disabled.
    */
   virtual void enableListeners() PURE;
+
+  /**
+   * Used by ConnectionHandler to manage listeners.
+   */
+  class ActiveListener {
+  public:
+    virtual ~ActiveListener() = default;
+
+    virtual uint64_t listenerTag() PURE;
```

Comment:
Can you add doxygen style comments for the functions here and below?

Reply:
done